doc/nrf/known_issues.rst

```diff
@@ -430,14 +431,14 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-8-0
+.. rst-class:: v1-9-0 v1-8-0
 
 KRKNWK-11720: NCP communication gets stuck
   The communication between the SoC and the NCP Host sometimes stops on the SoC side.
   The device neither sends nor accepts incoming packets.
   Currently, there is no workaround for this issue.
 
-.. rst-class:: v1-8-0
+.. rst-class:: v1-9-0 v1-8-0
```

Comment:
This issue is fixed in NCS v1.9
FYI: @maciejpietras

Reply:
Fixed